.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: node_js
```

Comment:
this is outdated file, please discuss with me

Reply:
still needs updates

Reply:
Done